psi4/src/psi4/libfock/apps.cc

```diff
@@ -50,6 +50,7 @@
 #include "psi4/libpsi4util/process.h"
 
 #include <algorithm>
+#include <functional>
```

Comment:
I might have missed it, but it doesn't look like we need functional here.

Reply:
`std::greater` is defined in `<functional>`. In the standard library of GCC, it is somehow included indirectly, but for MSVC it has to be explicit.